pkg/provider/notebook_resource_test.go

```diff
@@ -167,7 +180,7 @@ resource "datarobot_use_case" "test" {
 
 resource "datarobot_notebook" "test" {
   file_path = "%s"
-  use_case_id = [datarobot_use_case.test.id]
+  use_case_id = datarobot_use_case.test.id
```

Comment:
🤦 Stupid AI hallucinated this was an array randomly on an agentic run. I thought I fixed it everywhere. Thanks!